index.d.ts

```diff
@@ -0,0 +1,54 @@
+type Replacer = (key: string, value: any) => void;
+
+interface Options {
+	/**
+	 * Indentation as a string or number of spaces. Pass in null for no formatting.
+	 */
+	indent?: string | number | null;
+	/**
+	 * Detect indentation automatically if the file exists.
+	 */
+	detectIndent?: boolean;
+	/**
+	 * Sort the keys recursively. Optionally pass in a compare function.
+	 */
+	sortKeys?: boolean | ((a: string, b: string) => number);
+	/**
+	 * Passed into JSON.stringify.
+	 */
+	replacer?: Replacer | Array<number | string>;
+	/**
+	 * Mode used when writing the file.
+	 */
+	mode?: number;
+}
+
+/**
+ * Stringify and write JSON to a file atomically
+ *
+ * Creates directories for you as needed.
+ *
+ * @returns {void}
+ * @example
+ * import * as writeJsonFile from 'write-json-file';
+ *
+ * writeJsonFile.sync('foo.json', {foo: true});
+ * console.log('done');
+ */
+export function sync(filepath: string, data: any, options?: Options): void;
+
+/**
+ * Stringify and write JSON to a file atomically
+ *
+ * Creates directories for you as needed.
+ *
+ * @returns {Promise<void>}
```

Comment:
This is moot as it has no description and therefore is better documented by the type definition itself.

Reply:
👍 yea it's pretty pointless, it is defined by the Types